base/App/Home/HomeScreen.swift

```diff
@@ -17,7 +17,7 @@
 import SwiftUI
 
 struct HomeScreen: View {
-    @ObservedObject var settings: AppSettings = AppSettings.shared
+    @StateObject var settings: AppSettings = AppSettings.shared
```

Comment:
I think this makes sense for Screen ViewModels if needed... but who owns the singleton of AppSettings? Is this correct to use StateObject for a singleton value?